### ISSUE TITLE
Setup

### ISSUE THREAD

**Issue by sfle298201** [joshua-d-miller/macOSLAPS#100]
I have Active Directory, do we have to actually create a local account to use MacOS laps on each machine?

**Comment by crsleeth** [comment]
Yes. Or the local admin account needs to be created through automation/MDM. Be sure whatever setup you go with gives the account a Secure Token.

**Comment by joshua-d-miller** [comment]
@crsleeth is correct @sfle298201. You would either use MDM to create the administrator account to be used with LAPS or use a too lot create the administrator account on the device. When macOS devices are bound to AD they will write the password to the same field in AD as Windows devices do.
